scripts/lighthouse/index.js

```diff
@@ -0,0 +1,40 @@
+const stoplight = res => (res >= 90 ? '🟢' : res >= 75 ? '🟠' : '🔴');
+const normalizeScore = res => Math.round(res * 100);
+const formatScore = res => {
+  const normalizedScore = normalizeScore(res);
+  return `${stoplight(normalizedScore)} ${normalizedScore}`;
+};
+
+/**
+ * `core` is in scope from https://github.com/actions/github-script
+ * This is a CJS formatted file because using ESM here requires setting `type: module` in the package.json and that broke out site capabilities
```

Comment:
i tried ESM and it caused a postcss error

Reply:
WHich package.json? Of who? And why would PostCSS fail? O.o

Reply:
And to be honest, more than fine by adding "type" module at our package.json. We do prefer ESM anyways.

Reply:
i'll dig up the build that had it failing on my fork - maybe you can help me troubleshoot

Reply:
https://github.com/bmuenzenmeyer/nodejs.org/actions/runs/6632510994/job/18018374651#step:8:32

i will try messing with the postcss.config.js file and re-introduce this